docs/.vuepress/theme/components/Sidebar.vue

```diff
@@ -87,14 +145,14 @@ function resolveOpenGroupIndex(route, items) {
   top 7.5em
   right 0
   width $sidebarWidth
-  height 80vh
+  height calc(100vh - 80px)
```

Comment:
Like here, can we not just do something like `calc(100vh - 515px);` (`515px` being the height of the footer - or whatever is actually is - when the sidebar is present) vs. all the other JS within `mount()`?

Reply:
fixed this using position sticky. I'd be very keen, however, to avoid having to do calc() with the header height at all, let alone incorporating the footer height. It helps us to separate concerns